tests/cypress/support/welcome_and_header.js

```diff
@@ -117,5 +128,10 @@ export const userMenu = {
         cy.get('[data-test="user-dropdown"]').click()
         cy.get('#configure').should('not.exist')
         cy.get('#logout').should('not.exist')
+        cy.request('https://localhost:3000/multicloud/common/configure').as('configureReq')
```

Comment:
If you run this against real cluster, the url won't be localhost:3000. So I think this will fail.